### PR TITLE
Canisters update

### PR DESCRIPTION
**Pull request by dvir001**
## About the PR
Canisters now worth selling for base price of 1000
Canisters price in cargo up by 1000
Removed carbon canister from cargo
Fixed the canister spawner to have all of them in it.
Ship prices adjusted.

## Why / Balance
Give canisters a base worth for selling them.

## How to test
N/A

## Media
N/A

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes
Ship prices should go up, but sell will balance it out back.

**Changelog**
:cl:
- tweak: Canisters base worth up to 1000 from 100, cargo prices adjusted by 1000.
- remove: Removed the carbon canister from cargo.
- tweak: Some ships prices adjusted up by 2000~.
- tweak: All canister types can be found including Trit and Frezon in salvage and dungeons loot.

### CODE REVIEW

**Comment by dvir001** [comment]
Waiting for ship price test to make changes.

**Comment by whatston3** [comment]
If this is going through, should really consider putting #2330 in.

File: Resources/Prototypes/_NF/Shipyard/spirit.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@
   id: Spirit
   name: FSB Spirit
   description: A tiny medical search and rescue shuttle, as nimble as it is cramped. Running costs guarenteed* to be 5% lower than competing models!
```
Comment (whatston3) [inline Resources/Prototypes/_NF/Shipyard/spirit.yml:14]: Not really related, but while we're here...
```suggestion
  description: A tiny medical search and rescue shuttle, as nimble as it is cramped. Running costs guaranteed* to be 5% lower than competing models!
```

**Review by whatston3 — APPROVED** [review]
Looking through the YAML, changes seem reasonable.  From some rough napkin math, with the changes in #2421, at market rates,
most canisters are at ~$1.2K
plasma cans at ~$1.6K (~$5K without gas price changes)
tritium cans at ~$6K
frezon cans at ~$7K

Seems reasonable to me, more incentive for salvagers to salvage wrecks.

**Comment by dvir001** [comment]
Frick
```Arbitrage possible on Stasis. Minimal price should be 55293.632623762416, 4.999995% over the appraise price (52660.6048903007).
Assert.That(vessel.Price, Is.AtLeast(idealMinPrice))
  Expected: greater than or equal to 55293.6326237624[16](https://github.com/new-frontiers-14/frontier-station-14/actions/runs/11872499574/job/33086278004?pr=2412#step:9:17)d
  But was:  55000```

**Comment by whatston3** [comment]
Had a thought, just noting it here:

If you wanted to keep existing values the same, but increase values for salvaged canisters, you could make a distinction between new cans, equipped on ships and bought from cargo (thinner, crappy construction, cheaper to buy) and older cans found on salvage (beefier construction, higher max pressure?, higher value).

However: how would you differentiate the two visually, if at all?

**Comment by dvir001** [comment]
> Had a thought, just noting it here:
> 
> If you wanted to keep existing values the same, but increase values for salvaged canisters, you could make a distinction between new cans, equipped on ships and bought from cargo (thinner, crappy construction, cheaper to buy) and older cans found on salvage (beefier construction, higher max pressure?, higher value).
> 
> However: how would you differentiate the two visually, if at all?

Just leave them all the same im not getting bullied again for small canisters :(

**Comment by dvir001** [comment]
This is good to go, if we merge this consider also the 30k start change.

**Comment by dvir001** [comment]
Going to merge this before the gas changes, later today.

**Comment by dvir001** [comment]
Ok going to merge this, this will break some new ships pricing but nothing crazy to fix.

**Pull request merged**